src/utils/api.ts

```diff
@@ -345,94 +345,94 @@ export async function validateUsername(
  * ILS account.
  */
 export async function callPatronAPI(
+  res,
```

Comment:
I'd prefer not to pass `res` in because now the function is coupled with this object and how it returns data is tied to the server. Does it make it easier to return the response if this is passed rather than the object?

Reply:
I could refactor so the method returns the JSON and the calling method does the res.status setting.

Reply:
If it's a small and relatively quick update you can do before you leave, do it. Otherwise, let's do that next month.